### PR TITLE
Basic menu for the editor

### PR DESCRIPTION
**Pull request by jolive3000**

### CODE REVIEW

**Comment by FFFiend** [comment]
I had a cursory look over the changes so far, and I'm super curious with how you're handling the panel switching, with passing objects into the call to ActionListener. I say that because I myself am struggling right now to get the panels to switch the way we want, so I might need to draw some inspiration from you :DD

You could add some basic tests for the panels, but since they aren't headless (i.e. they require mouse and keyboard input) you could just test the button presses, or the ActionListener events and test their output or have them output something to the console and check that to ensure that they're working as intended.

**Comment by jolive3000** [comment]
Unfourtunatley Owais there is no panel switching, it is just creating new JFrames without deleting the old ones. 
I was going to add code to close a JFrame on creation of a new, but I didn't implement that since I think your idea is cleaner

**Comment by FFFiend** [comment]
Ah alright, I think someone helped me out with that on Piazza so I'll take a look and get back to you on how you could do the panel switching as well

**Review by FFFiend — DISMISSED** [review]

File: src/main/java/use_cases/custom_game/custom_game_file_management/CustomGameValidator.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package use_cases.custom_game.custom_game_file_management;
+
+import adapters.custom_game.custom_game_file_adapters.EditorTile;
```
Comment (FFFiend) [inline src/main/java/use_cases/custom_game/custom_game_file_management/CustomGameValidator.java:3]: no violations of Clean Architecture either :D

**Review by tseamuscorlett — APPROVED** [review]
Awesome!

As you commented, there are things to be added / removed as we finalize the projects
But I'll approve it so we can merge

**Pull request merged**